linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/router_server.js

```diff
@@ -0,0 +1,28 @@
+var RouterServer = (function() {
+  // Skeletal implementation to demonstrate functionality of RouterController
+
+  function initializeServerContainers(servers, $serverEl) {
```

Comment:
This just stems from me not knowing javascript -- why are some function parameters prefixed with "$" and not others?  `servers` vs `$serverEl` etc.?  Is that to indicate that serverEl is a fully extended object?